frontend/DataView/DataView.js

```diff
@@ -306,6 +323,13 @@ var styles = {
     top: 4,
   },
 
+  toggler: {
+    marginLeft: -12,
```

Comment:
if we're already doing `position: absolute`, might as well just do `left: -12` here, to align with `top: -1`. And we probably don't need `paddingRight: 3`?
In fact, I don't think the `padding` will have any effect on a checkbox